### PR TITLE
Portfolio-setup: Mobile Version

### PR DESCRIPTION
**Pull request by 3nityGitHub**
## In this PR:

- I added the work section and all the relevant images,
- I added the about me section

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Ouch-gifs-38715704-365-199

You have submitted a project that's not sufficiently following the [updated readme template](https://github.com/microverseinc/readme-template). Kindly follow the template to include the table of content and make sure they're links to parts of the readme.

You seem to have some table of content, but the items are not links and some sections are missing from your readme.
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -41,4 +41,140 @@ <h2 class="h2_title">I am a Software Developer.</h2>
...
+      <img class="img_placeholder" src="image folder/ImgPlaceholder.png" alt="frame">
+    <h3 class="multi_story">Multi-Post Stories</h3>
+    <p class="recent_work_text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    <ul class="recent_work_language flex">
+      <li class="work_language flex">css</li>
+      <li class="work_language flex">html</li>
+      <li class="work_language flex">Bootstrap</li>
+      <li class="work_language flex">Ruby</li>
+    </ul>
+    <button class="see_prjt_buttn flex">See Project</button>
```
Comment (RiyaBulia12) [inline index.html:62]: - [ ] It looks like you have not removed the border from all the `See Projects` buttons. Please kindly fix the button design to make it similar to the Figma template. Please try to stick with your selected design template for better UI and consistency. 👍🏽 😃 

| Figma Design | Your Design |
|------------------- | ------------------- |
| Figma_1 | Your |

File: index.html [hunk 2]
```diff
@@ -41,4 +41,140 @@ <h2 class="h2_title">I am a Software Developer.</h2>
...
+  </div>
+
+ </section>
+
+ <section class="About_me">
+  <div class="about_me_container">
+    <h2 class="about_me_title">About me.</h2>
+    <p class="about_me_content">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+    <button class="get_resume flex">Get My Resume</button>
```
Comment (RiyaBulia12) [inline index.html:143]: - [ ] It looks like you have not removed the border from all the `See Projects` buttons. Please kindly fix the button design to make it similar to the Figma template. Please try to stick with your selected design template for better UI and consistency. 🙂

**Review by RiyaBulia12 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi  @3nityGitHub 👋, 

Good job! 🎉 🎉 🎉  There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight 🎉
- No linter errors :heavy_check_mark:
- Correct GitHub flow :heavy_check_mark:
- Meaningful commit messages ✔️
- License setup correctly  ✔️ 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._ 🙂 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -41,4 +41,140 @@ <h2 class="h2_title">I am a Software Developer.</h2>
...
+                </li>
+                <li class="language">RSpec</li>
+                
+                <li class="language">CapyBara</li>
+                <li class="language">Selenium</li>
+                <li class="language">JBootstrap</li>
+                <li class="language">Ruby on Rails</li>
+            </ul>
+             <ul class="skill_list flex">
+                <li class="Language_skill2"><img class="border_line" src="image folder/Rectangle57.png" alt="rectangle_icon">  Skills
```
[the remark this answers is not in the document]
Comment (3nityGitHub) [reply index.html:169]: @RiyaBulia12 Thanks for your review, but I still cannot understand what exactly you want me to do here, the my design looks close to the figma design as possible. Kindly revert asap.

Regards.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @3nityGitHub , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions
### Your logo should be wrapper by a an `a` tag because if should have a link to the home page. This is best practice for website particularly to improve user experience. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (RiyaBulia12) [reply index.html:169]: Hi @3nityGitHub, Sorry for the delayed response. It looks like I used the wrong screenshot for comparison. Sorry again for the confusion. I was suggesting you align all the headings in the middle, especially Skills. Something like this in the image below:

Screenshot 2022-12-03 at 4 26 02 PM

**Pull request merged**